src/hazelcore/Action_Exp.re

```diff
@@ -3233,6 +3361,8 @@ and ana_perform_operand =
       }
     }
 
+  | (Construct(SCloseBraces), _) => Failed
```

Comment:
same as above comment

Reply:
> Looks pretty good, just a few small things to respond to / revise. Also, can you add support for `]` when in a list type like `[Int]`? (We'll have list literals in a future pull too, so this can set up for that as well.)

Done!